### DISCUSSION TITLE
About Permissions

### DISCUSSION THREAD

**Discussion by Nootaku** [tidal-music/discussions#13] [General]
Hello,

I have been using the web-player for a while now and have created an Electron wrapper for my Fedora machine.
Since I like the service I wanted to download and install the app on my smartphone.

I'm generally interested in the privacy of applications that go on my phone so I always have a look at trackers and permissions before installing. I must stay that the Tidal application report looks ... sub-par in that regard ([exodus-report](https://reports.exodus-privacy.eu.org/en/reports/com.aspiro.tidal/latest/)).

I admit that this is still better than Spotify.

I have no real experience with developing mobile applications, but I was wondering if it wouldn't be possible to remove some of those critical permissions. For example, why does Tidal need to have the right to record audio ?

**Comment by stefanmoro** [comment]
Hey @Nootaku, and welcome to the forum.

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com) (TIDAL API, open source projects etc).

As this question relates to one of our existing products, I will kindly ask you to reach out to our [main TIDAL support](https://support.tidal.com/).

Thanks
